static-analysis/assert-is-list.php.disabled

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertString;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @param mixed $value
+ *
+ * @return array<int, mixed>
```

Comment:
Not sure if psalm can restrict an `array<mixed>` to an `array<int, mixed>`, or `array<Foo>` to `array<int, Foo>` here